Resources/Server Info/Guidebook/Law/Trials.xml

```diff
@@ -0,0 +1,15 @@
+﻿<Document>
+# Trial Procedure
+
+On Nyanotrasen, trials are conducted in the form of an [color=#a4885c]arbitration.[/color] This means that:
+
+- You are [color=#a4885c]not required[/color] to have a [color=#a4885c]lawyer[/color].
+- You need a trial only when you are accused of a [color=#a4885c]Capital Crime[/color].
```

Comment:
> You need a trial only when you are accused of a Capital Crime.

This sounds like you have to do a trial for Capital Crimes even if the defendant doesn't want one. In the Space Law it's: "If the defendant is accused of a Capital Crime, they are entitled to trial"

Reply:
**This is something I will actually transfer into full Space Law.** Trials are meant to be super quick, so there's no reason for Security to not do them. If a Capital Crime is charged, trial must happen, and that's even if the prisoner confesses, which is when it's made even speedier. It's literally three people in a room with the HoS passing judgement upon hearing an explanation of what happened. If we can't push Sec to RP that little, why bother with an MRP server. I can't dumb this down any more.